test/stage2/compile_errors.zig

```diff
@@ -1,8 +1,51 @@
 const TestContext = @import("../../src-self-hosted/test.zig").TestContext;
+const std = @import("std");
+
+const ErrorMsg = @import("../../src-self-hosted/Module.zig").ErrorMsg;
+
+const linux_x64 = std.zig.CrossTarget{
+    .cpu_arch = .x86_64,
+    .os_tag = .linux,
+};
 
 pub fn addCases(ctx: *TestContext) !void {
     // TODO: re-enable these tests.
     // https://github.com/ziglang/zig/issues/1364
+    ctx.addZIRError("call undefined local", linux_x64,
+        \\@noreturn = primitive(noreturn)
+        \\
+        \\@start_fnty = fntype([], @noreturn, cc=Naked)
+        \\@start = fn(@start_fnty, {
+        \\  %0 = call(%test, [])
+        \\})
+    , &[_][]const u8{"5:13:unrecognized identifier: %test"});
```

Comment:
This should look like, e.g. `5:13: error: unrecognized identifier: %test`, because that's what zig compile errors look like. We're also going to need to be able to test notes.

Reply:
Gotcha; it's simple enough to just check that we see " error: " and then slice past it.